xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -255,11 +256,18 @@ public void setUp() throws IOException {
     channel =
         cleanupRule.register(InProcessChannelBuilder.forName(serverName).directExecutor().build());
 
+    Bootstrapper.BootstrapInfo bootstrapInfo =
+        new Bootstrapper.BootstrapInfo(
+            Arrays.asList(
+                new Bootstrapper.ServerInfo(
+                    "", InsecureChannelCredentials.create(), useProtocolV3())),
```

Comment:
nit: do not just use an empty string, even though the string is actually used. Try make test input as realistic as possible.